src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+                - --collectors=persistentvolumeclaims
+                - --collectors=persistentvolumes
+                - --collectors=poddisruptionbudgets
+                - --collectors=pods
+                - --collectors=replicasets
+                - --collectors=replicationcontrollers
+                - --collectors=resourcequotas
+                - --collectors=secrets
+                - --collectors=services
+                - --collectors=statefulsets
+                - --collectors=storageclasses
+                - --collectors=volumeattachments
+                image: docker.io/bitnami/kube-state-metrics:1.9.8-debian-10-r0
+                imagePullPolicy: IfNotPresent
+                livenessProbe:
+                  failureThreshold: 6
+                  httpGet:
+                    path: /healthz
+                    port: http
+                    scheme: HTTP
+                  initialDelaySeconds: 120
+                  periodSeconds: 10
+                  successThreshold: 1
+                  timeoutSeconds: 5
+                name: kube-state-metrics
+                ports:
+                - containerPort: 8080
+                  name: http
+                  protocol: TCP
+                readinessProbe:
+                  failureThreshold: 6
+                  httpGet:
+                    path: /
+                    port: http
+                    scheme: HTTP
+                  initialDelaySeconds: 30
+                  periodSeconds: 10
+                  successThreshold: 1
+                  timeoutSeconds: 5
+                resources: {}
+                terminationMessagePath: /dev/termination-log
+                terminationMessagePolicy: File
+              dnsPolicy: ClusterFirst
+              restartPolicy: Always
+              schedulerName: default-scheduler
+              securityContext:
+                fsGroup: 1001
+                runAsUser: 1001
+              serviceAccount: ksm-kube-state-metrics
+              serviceAccountName: ksm-kube-state-metrics
+              terminationGracePeriodSeconds: 30
+              volumes:
+              - emptyDir: {}
+                name: tmpfs-data
+              - emptyDir: {}
+                name: tmpfs-user-data
+              - emptyDir: {}
+                name: tmpfs-tmp
+              - emptyDir: {}
+                name: tmpfs-cache
+        ```
+  
+  </Collapser>
+</CollapserGroup>
+
+3. Create a `ClusterRoleBinding`, or add to a previously created one the `ServiceAccount` of the application that is going to be monitored. All the workloads may share the same `ClusterRoleBinding`, but the `ServiceAccount` of each one must be added to it.
+4. Create a secret containing the New Relic license key. Each namespace needs its own secret.
+
+#### Configuration [#config-manual]
+
+When adding the manifest of the sidecar agent manually, you can use any agent configuration option to configure the agent behavior.
+**NEED A LINK HERE**
```

Comment:
https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/

Reply:
@paologallinaharbur, I added this link: For help, see [Infrastructure agent configuration settings](/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/).